skore/src/skore/sklearn/cross_validation/cross_validation_reporter.py

```diff
@@ -124,6 +124,7 @@ def __init__(self, *args, **kwargs):
         self.estimator = args[0] if len(args) >= 1 else kwargs.get("estimator")
         self.X = args[1] if len(args) >= 2 else kwargs.get("X")
```

Comment:
`X` seems to be mandatory?

```suggestion
        self.X = args[1] if len(args) >= 2 else kwargs["X"]
```

Reply:
The thing is, I would rather `sklearn.cross_validate` be the one to issue the error if X was not passed: with your change, we'd have to add an error message that X wasn't passed.